sites/docs/pages/getting-started.md

```diff
@@ -0,0 +1,163 @@
+---
+sidebar_position: 3
+hide_table_of_contents: false
+title: Build your first dashboard
+description: A 10 minute guide to building your first dashboard in Evidence.
+---
+<div id="objectives" class="alert svelte-17118v7">
+<strong>By the end of this 10 minute guide, you will be able to:</strong>
+<ul>
+   <li>Create and edit a page in Evidence</li>
+   <li>Write a query on demo data</li>
+   <li>Create a table</li>
+   <li>Create a chart</li>
+   <li>Connect a new CSV data source</li>
+</ul>
+</div>
+
+
+<Alert status="info">
+Casey note: I like how the Rails tutorial calls out what you'll achieve, and adds these satisfying green checkmarks. I think it makes the reader feel good and excited. Baked my own CSS for this, happy to change to your suggestions
```

Comment:
One more thought - I see that the intended usage of the Blue info Alert box is to call people's attention to things that have recently changed.

<img width="986" alt="image" src="https://github.com/evidence-dev/evidence/assets/16248601/1a4991fa-bfe4-4122-96a0-6278f0ae5121">

My intention with the grey boxes in this tutorial is less so that, and more of an "interesting additional context that you may optionally explore" kind of thing. This seems to match up with your default alert component and I think it's literally just the padding that bothers me. 

<img width="780" alt="image" src="https://github.com/evidence-dev/evidence/assets/16248601/386f0d8c-be26-46cb-8845-89f8104a255a">

I've written each grey box as an Alert as well so you can compare them side by side. If you don't see enough of a visual difference to justify making a new component, I'm happy to swap them over:

<img width="805" alt="image" src="https://github.com/evidence-dev/evidence/assets/16248601/d93d92fd-6b5f-4382-9bbc-1cfa14db29eb">